### ISSUE TITLE
Usage in other projects?

### ISSUE THREAD

**Issue by DPS2004** [balpha/playdate-types#2]
Hello, I just finished making a plugin for [ZeroBrane Studio](https://studio.zerobrane.com/) that adds support for the Playdate SDK. It allows launching the simulator and installing to hardware directly from the editor, and adds autocomplete for the API. To get autocomplete to work, I wrote a script that converts __types.lua to the autocomplete format ZeroBrane uses. 

Since I'm probably not the only person using ZeroBrane for Playdate development, I figure it would probably be a useful tool to share with others. Would I be allowed to share the conversion script and/or the generated API definition file? I'm not sure if this falls under "using" the file as described in the license, and if you don't want it shared or made public I totally understand and respect that. Either way, thank you for making such a useful resource!

**Comment by balpha** [comment]
I'm happy you find it useful! And personally I don't have any objection.

The thing is this: Panic releases the Playdate SDK under a proprietary license that doesn't allow sharing of things like this without explicit permission. And `__types.lua` is based directly on the [documentation](https://sdk.play.date/2.5.0/), which is also covered by that license.

That's why I [asked for permission](https://devforum.play.date/t/lua-type-annotation-file-for-the-sdk/12165/1) on the forum and they eventually (after 3.5 months) [gave me that permission](https://devforum.play.date/t/lua-type-annotation-file-for-the-sdk/12165/28).

Honestly, I think they would probably be fine -- other people have shared similar projects without jumping through hoops, and I don't think Panic actually objected to any of them. After all, they should be happy about widely available developer tooling for their platform.

However, I myself wanted to be on the legally safe side and thus only made this repo public once I had permission.

Long story short: As far as my own work is concerned, feel free to use it (attribution appreciated!). As far as Panic's ownership and license of the content is considered, you'll have to ask them (or whatever you decide).

**Comment by DPS2004** [comment]
Got it. I think if there are other shared projects that Panic hasn't objected to, I'll make it public. Should now be available [here](https://github.com/DPS2004/playdate-zerobrane)!
